extensions/azurecore/src/account-provider/azureAccountProviderService.ts

```diff
@@ -146,9 +162,47 @@ export class AzureAccountProviderService implements vscode.Disposable {
 			}
 			let simpleTokenCache = new SimpleTokenCache(tokenCacheKey, this._userStoragePath, noSystemKeychain, this._credentialProvider);
 			await simpleTokenCache.init();
+			const cachePath = path.join(this._userStoragePath, './cache.json');
+			let platform = os.platform();
+			let persistence;
+			const serviceName = 'azuredatastudio';
```

Comment:
should these be globally defined some where?  seems like embedded directly in the function is a bit obsure.